suite-native/module-settings/src/components/ColorSchemePickerItem.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { ColorValue, TouchableOpacity } from 'react-native';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { colorVariants, ThemeColorVariant } from '@trezor/theme';
+import { Box, Text } from '@suite-native/atoms';
+
+type ColorSchemePickerItemProps = {
+    isSelected?: boolean;
+    onPress: () => void;
+    colorScheme: ThemeColorVariant;
+};
+
+const pickerItemWrapperStyle = prepareNativeStyle<{ isSelected: boolean }>(
+    (utils, { isSelected }) => ({
+        backgroundColor: utils.colors.gray100,
+        borderRadius: utils.borders.radii.medium,
+        height: 114,
+        flex: 1,
+        paddingTop: 33,
+        borderWidth: utils.borders.widths.medium,
+        borderColor: isSelected ? utils.colors.green : utils.colors.gray100,
+    }),
+);
+
+type PickerItemDotStyleProps = {
+    backgroundColor: ColorValue;
+    isFirstItem: boolean;
+};
+const pickerItemDotStyle = prepareNativeStyle<PickerItemDotStyleProps>(
+    (utils, { backgroundColor, isFirstItem }) => ({
+        width: 26,
+        height: 26,
+        backgroundColor,
+        borderRadius: utils.borders.radii.round,
+        extend: {
+            condition: !isFirstItem,
+            style: {
+                marginLeft: -utils.spacings.small,
+            },
+        },
+    }),
+);
+
+const textStyle = prepareNativeStyle(utils => ({
+    alignSelf: 'center',
+    paddingTop: 23,
+    paddingBottom: utils.spacings.small,
+    textTransform: 'capitalize',
+}));
+
+export const ColorSchemePickerItem = ({
+    onPress,
+    colorScheme,
+    isSelected = false,
+}: ColorSchemePickerItemProps) => {
+    const { applyStyle } = useNativeStyles();
+    return (
+        <TouchableOpacity
+            onPress={onPress}
+            style={applyStyle(pickerItemWrapperStyle, { isSelected })}
+        >
+            <Box flexDirection="row" justifyContent="center">
+                <Box
+                    style={applyStyle(pickerItemDotStyle, {
+                        backgroundColor: colorVariants[colorScheme].gray400,
+                        isFirstItem: true,
+                    })}
+                />
+                <Box
+                    style={applyStyle(pickerItemDotStyle, {
+                        backgroundColor: colorVariants[colorScheme].gray500,
+                        isFirstItem: false,
+                    })}
+                />
+                <Box
+                    style={applyStyle(pickerItemDotStyle, {
+                        backgroundColor: colorVariants[colorScheme].black,
+                        isFirstItem: false,
+                    })}
+                />
+            </Box>
+            <Text style={applyStyle(textStyle)} color={isSelected ? 'green' : 'gray600'}>
+                {colorScheme}
```

Comment:
I would probably add one more prop like `label` because it could be little bit different that our name in code, and for example dark could be even translated in future etc.

Reply:
Yeah but we could always just change it in future? I see what you're saying. But it might not be changed and could be just used like this imo. I'm up for leaving it and if it needs to change, it's easy to add label when it's needed.

Reply:
Lets do that later